### ISSUE TITLE
[Quest][Azuremist Isle][Ammen Vale] - {The Emmiter} - Missing quest NPC event - holographic message

### ISSUE THREAD

**Issue by ashphoenix92** [dalaranwow/dalaran-wow#4157]
Last part of the quest-chain leads to NPC Zhana before starting next quest {Travel to Azure Watch]. After talking to her quest event is currently missing. The event that happens afterwards:

<Image of Technician Dyvuun appears on the emitter.>
Image of Technician Dyvuun says: By the seven Ata'mal crystals! Others have survived the crash! Zhanaa, is that you? It is so good to see you again!
Image of Technician Dyvuun says: And who do we have here? No doubt th... crzzzk ...ade this communication I give you greetings fro... psshzzzk... Azure Watch.
Image of Technician Dyvuun says: Judging by your signal, we put you... ust southeast of the larger part of the island we crashed onto. Cross the Crystalbrook River to ge... kshhhhk zzzt. Sorry, there's a lot of stat...
Technician Zhanaa says: This is great news! Your efforts have indeed saved us, <name>!

http://www.wowhead.com/quest=9312/the-emitter

**Comment by Rushor** [comment]
similar event was added here https://github.com/dalaranwow/dalaran-wow/issues/4082

**Comment by Rushor** [comment]
oh i already fixed that, duplicate of https://github.com/dalaranwow/dalaran-wow/issues/2388 it will be tested on PTR.
